qa/tests/test_about_page.py

```diff
@@ -0,0 +1,13 @@
+from qa.PageObjects.AboutPage import AboutPage
+from qa.utilities.BaseClass import BaseClass
+
+
+class TestAboutPage(BaseClass):
+
+    def test_selenium_about(self):
+        self.driver.get("http://127.0.0.1:5000/about")
```

Comment:
Move URLs to one place, e.g. env variables or config.py file with global used variables